pkgs/top-level/aliases.nix

```diff
@@ -368,51 +368,49 @@ mapAliases {
   # Added 2018-07-16 preserve, reason: forceSystem should not be used directly in Nixpkgs.
   forceSystem = system: _: (import self.path { localSystem = { inherit system; }; });
 
-  ### _ ###
+  # When the nixops_unstable alias is removed, nixops_unstable_minimal can be renamed to nixops_unstable.
```

Comment:
Let's remove this outdated comment in a commit before (again, to make the sorting commit entirely reproducible)